awslabs/amazon-timestream-tools
# Enable Code coverage for Flink Connector and report Unit Rests

## Summary
Provide code coverage information on the project README.md file

## Description
Add GIT Action that runs Flink Connector Unit tests and generates a test coverage badge based on the results.
On the readme, the path to use and show the badge has been added

## Tests performed/created
### Manual verification:
1. Make changes in the code
2. Push changes 
3. Merge PR
4. Verify that git action has been executed
5. Verify that the badge with the latest test coverage result has been generated


## Additional Reviewers
@alexeytemnikov
